lib/request.js

```diff
@@ -8,9 +8,18 @@ var request = require('request');
 require('sugar');
 
 var url     = require('url'),
-    util    = require('util');
+    util    = require('util'),
+    path    = require('path');
 
-module.exports = function (protocol, host, version, token) {
+module.exports = function (info) {
```

Comment:
Decided against passing even more arguments, instead passing a map `{}`, bit like the request module itself does. 
Hope this is ok!